kornia/augmentation/container/augment.py

```diff
@@ -498,15 +498,22 @@ def __call__(
             if len(inputs) == 1 and isinstance(inputs[0], dict):
                 original_keys, in_data_keys, inputs, invalid_data = self._preproc_dict_data(inputs[0])
             else:
-                in_data_keys = kwargs["data_keys"] if "data_keys" in kwargs else self.data_keys
+                in_data_keys = kwargs.get("data_keys", self.data_keys)
             data_keys = self.transform_op.preproc_datakeys(in_data_keys)
 
-            if len(data_keys) > 1 and data_keys.index(DataKey.INPUT):
+            if len(data_keys) > 1 and DataKey.INPUT in data_keys:
```

Comment:
I'm assuming this should be checking if `DataKey.INPUT` is in data_keys? Otherwise, we would only enter the if block if `len(data_keys) > 1` and if INPUT is not the first datakey.